src/activespeakerpolicy/DefaultActiveSpeakerPolicy.ts

```diff
@@ -1,11 +1,44 @@
-// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// Copyright 2019-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
 import ActiveSpeakerPolicy from './ActiveSpeakerPolicy';
 
 export default class DefaultActiveSpeakerPolicy implements ActiveSpeakerPolicy {
+  /**
+   * Map of attendeeIds to their active speaker score values
+   */
   private volumes: { [attendeeId: string]: number } = {};
 
+  /** Creates active speaker policy with speakerWeight, cutoffThreshold, silenceThreshold, and takeoverRate.
+   *
+   * @param speakerWeight
+   * Number used to calculate new active speaker score for current attendee
+   * ```js
+   * Formula:
+   * updatedCurrentAttendeeScore = currentAttendeeExistingScore * speakerWeight + currentReceivedVolume * (1 - speakerWeight)
+   * ```
+   *
+   * @param cutoffThreshold
+   * Threshold number compared with updated active speaker score.
+   * If the updated active speaker score is less than this threshold value,
+   * the updated score is returned as 0, else the updated score is returned.
+   *
+   * @param silenceThreshold
+   * Threshold number compared with current received volume.
+   * While calculating the new active speaker score, if the current received
+   * volume is less than this threshold value, the current received volume is considered as 0,
+   * else 1.
+   *
+   * @param takeoverRate
+   * Number used to calculate other attendees active speaker score, other than the current attendee.
```

Comment:
`other attendees` => `other attendee's` or `other attendees'`

Reply:
Yes, corrected it thanks for catching.